dnf-json

```diff
@@ -24,7 +24,7 @@ except IndexError:
 
 base = dnf.Base()
 
-base.conf.cachedir = "./dnf-cache"
+base.conf.cachedir = "/var/cache/osbuild-composer/dnf-cache"
```

Comment:
Let's just use the systems' normal cache.

Reply:
What do you mean by systems' normal cache?

Reply:
DNF's cache is usually in `/var/cache/dnf` (but can be configured, of course). I'd say let's just use whatever is the configured dnf cache on the system instead of having our own. This way, it can be shared.

In other words: please delete that line :)

Reply:
The thing is that `/var/cache/dnf` is owned by root, but we're running osbuild-composer as non-root user.

Reply:
I was wrong, it works just right, I guess libdnf makes some kind of rights escalation... I've updated the PR.